src/modules/accounts/useCases/RegisterUser/RegisterUserController.spec.ts

```diff
@@ -29,4 +30,14 @@ describe('Register User (e2e)', () => {
 
     expect(userInDatabase).toBeTruthy()
   })
+
+  it('should return an error if validation fails', async () => {
```

Comment:
Do you think we should include tests for validation inside controller specs? I don't know if we need that.

Reply:
This test isn't for the validation itself, is just to make sure that the controller is actually validating the request and returning an client error if validation fails... I thought was i good idea ensure that we've this behavior with a test. But isn't a **crucial** point.

Reply:
Yeah, so I think we can keep this way.